themes/pzq/pzq.theme.sh

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env bash
+
+# Theme inspired on:
+#  - ys theme - https://github.com/ohmyzsh/ohmyzsh/blob/master/themes/ys.zsh-theme
+#  - rana theme - https://github.com/ohmybash/oh-my-bash/blob/master/themes/rana/rana.theme.sh
+#
+# by Ziqiang Pei<ziqiangpei@foxmail.com>
+
+# ----------------------------------------------------------------- COLOR CONF
+D_DEFAULT_COLOR="${white}"
+D_INTERMEDIATE_COLOR="${bold_white}"
+D_NUMBER_COLOR="${bold_blue}"
+D_USER_COLOR="${cyan}"
+D_SUPERUSER_COLOR="${red}"
+D_MACHINE_COLOR="${green}"
+D_DIR_COLOR="${bold_yellow}"
+D_GIT_DEFAULT_COLOR="${echo_white}"
+D_GIT_BRANCH_COLOR="${echo_cyan}"
+D_GIT_PROMPT_COLOR="${echo_red}"
+D_SCM_COLOR="${bold_yellow}"
+D_BRANCH_COLOR="${cyan}"
+D_CHANGES_COLOR="${white}"
+D_TIME_COLOR="${white}"
+D_DOLLOR_COLOR="${bold_red}"
+D_CMDFAIL_COLOR="${red}"
+D_VIMSHELL_COLOR="${cyan}"
+
+# ------------------------------------------------------------------ FUNCTIONS
+case $TERM in
+  xterm*)
+      TITLEBAR="\033]0;\w\007"
```

Comment:
```suggestion
      TITLEBAR="\[\033]0;\w\e\\\]"
```

- OSC terminated by BEL is deprecated. It is recommended to use the ANSI form terminated by ST (`\e\\`).
- We need to enclose it with `\[...\]` to let Bash know that the cursor will not be moved by the sequence.